apps/src/sites/studio/pages/code-studio.js

```diff
@@ -12,57 +12,57 @@ require("babel-polyfill");
 import $ from 'jquery';
 
 // Shim window.console to be safe in IE
-require('./consoleShim')(window);
+require('@cdo/apps/code-studio/consoleShim')(window);
 
 var Sounds = require('@cdo/apps/Sounds');
-var activateReferenceAreaOnLoad = require('./reference_area');
+var activateReferenceAreaOnLoad = require('@cdo/apps/code-studio/reference_area');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
 window.Radium = require('radium');
 
 // TODO (bbuchanan): Stop including these components in a global way, just
 //                   require them specifically where needed.
-require('./components/abuse_error');
-require('./components/report_abuse_form');
-require('./components/SendToPhone');
-require('./components/small_footer');
-require('./components/GridEditor');
-require('./components/Attachments');
+require('@cdo/apps/code-studio/components/abuse_error');
+require('@cdo/apps/code-studio/components/report_abuse_form');
+require('@cdo/apps/code-studio/components/SendToPhone');
+require('@cdo/apps/code-studio/components/small_footer');
+require('@cdo/apps/code-studio/components/GridEditor');
+require('@cdo/apps/code-studio/components/Attachments');
```

Comment:
My initial reaction is that this might be too restrictive, though I could be wrong about that.